app/src/main/java/org/simple/clinic/instantsearch/InstantSearchScreenKey.kt

```diff
@@ -5,11 +5,13 @@ import kotlinx.android.parcel.IgnoredOnParcel
 import kotlinx.parcelize.Parcelize
 import org.simple.clinic.navigation.v2.ScreenKey
 import org.simple.clinic.patient.businessid.Identifier
+import org.simple.clinic.scanid.PatientPrefillInfo
 
 @Parcelize
 data class InstantSearchScreenKey(
     val additionalIdentifier: Identifier?,
-    val initialSearchQuery: String?
+    val initialSearchQuery: String?,
+    val patientPrefillInfo: PatientPrefillInfo?
```

Comment:
Created a [story](https://app.clubhouse.io/simpledotorg/story/3639/resolve-coupling-between-scanid-and-instantsearch-packages?vc_group_by=day&ct_workflow=all) for this will resolve in another PR.